### DISCUSSION TITLE
starter pack cannot add labeler.

### DISCUSSION THREAD

**Discussion by itaru2622** [bluesky-social/atproto#2614] [General Discussion]
when I try to add labeler (xblock.aendra.dev) in starter pack via 'search', it said "nobody found"...
It is better to support labelers in starter pack, by some way.

**Comment by bnewbold** [comment]
We considered adding explicit support to labelers to the initial version of starter packs, but decided not to do it in this first iteration. The labeling ecosystem is relatively small and still developing compared to feed generators. We don't want to overwhelm small labeler efforts until the ecosystem has matured a bit. Similarly, we don't support ranking of labelers by most popular, etc.

Feed generators have been around more than a year, while labelers have only been around a couple months. I would expect that labelers will be more discoverable and integrated in due time.
